dev-docs/docusaurus.config.js

```diff
@@ -132,6 +132,9 @@ const config = {
 
         indexName: "YOUR_INDEX_NAME",
       },
+      tableOfContents: {
+        maxHeadingLevel: 4,
```

Comment:
since level 1 is unused, we generally have 3 toc levels to work with. Let's not hide level-4 from ToC. If the sections get too long an break, maybe we should rename as I did in WelcomeScreen.